site/content/en/docs/manual/advanced/filter.md

```diff
@@ -95,3 +118,81 @@ Click `Submit` to apply the filter.
 The `Cancel` button undoes the filter. The `Clear filter` button removes the filter.
 
 Once applied filter automatically appears in `Recent used` list. Maximum length of the list is 10.
+
+---
+
+## Sort and filter projects, tasks, jobs and cloud storage
+
+On the [projects](/docs/manual/advanced/projects/#projects-page), [tasks](/docs/manual/basics/tasks-page/),
+[jobs](/docs/manual/basics/jobs-page/), and [cloud storage](/docs/manual/basics/cloud-storages/) pages,
+you can use sorting and filters.
+
+### Sort by
+
+You can sort the project, task or job by the following parameters:
+- `ID` - ID project, task or job
+- `Assignee` - the user to whom the project, task or job is assigned
+- `Owner` - the user who owns the project, task, or job
+- `Status` - sort by status (`annotation`, `validation` or `completed`)
+- `Updated date` - time and date of last saved project, task or job
+- `Stage` - stage set on the task page
+- `State` - state set by a user assigned to the job
+- `Subset` - sort by subset (`test`, `train`, `validation` or custom subset
+)
+- `Mode` - sort by mode (`interpolation - video` or `annotation - images`)
+- `Dimension` - sort by dimension `2D` and `3D`,
+  read more in [creating an annotation task](/docs/manual/basics/creating_an_annotation_task)
+- `Task ID` - the ID of the task to which the job belongs
+- `Project ID` - the ID of the project containing the task to which the job belongs.
+- `Name` - name of task or project
+- `Task name` - the name of the task to which the job belongs
+- `Project name` - the name of the project containing the task or the task to which the job belongs.
+- `Provider type` - sort by provider (`AWS S3`, `Azure`, `Google cloud`)
+- `Credentials` - sort by credentials (`Key & secret key`, `Account name and token`, `Anonymous access`, `Key file`)
+- `Description` - sort by description
+- `Resource` - sort by resource (`bucket name` or `container name`)
+- `Display name` - sort by display name
```

Comment:
The same  comment about splitting into different groups.
The most of descriptions here (like `Description` - sort by description, `Name` - name of task or projects) seem to be totally useless.

Maybe define such formats:
```
Tasks: ID, assignee, owner, status, ...
Projects: ID, assignee, owner, status, ....
Cloud storages: Description, display name, .... 
```